Cargo.toml

```diff
@@ -16,8 +16,8 @@ keywords = ["gamedev", "amethyst", "physics"]
 [dependencies]
 amethyst_core = { version = "0.3.0" }
 amethyst_renderer = { version = "0.8.0" }
-rhusics-ecs = { version = "0.5", features = ["serde"] }
-rhusics-core = { version = "0.5", features = ["specs", "serde"] }
+rhusics-ecs = { version = "0.5", features = ["serde"], git = "https://github.com/rustgd/rhusics" }
```

Comment:
might be a better idea to release a version to crates.io for this.

Reply:
There are other changes coming soon